rails/config/routes.rb

```diff
@@ -188,10 +180,11 @@
     match '/signup' => 'users#new', :as => :signup, via: [:get, :post]
     match '/activate/:activation_code' => 'users#activate', :as => :activate, :activation_code => nil, via: [:get, :post]
     match '/forgot_password' => 'passwords#login', :as => :forgot_password, via: [:get, :post]
-    match '/forgot_password/email' => 'passwords#email', :as => :forgot_password_email, via: [:get, :post]
+    match '/forgot_password/email' => 'passwords#login', :as => :forgot_password_email, via: [:get, :post]
     match '/change_password/:reset_code' => 'passwords#reset', :as => :change_password, via: [:get, :post]
     match '/password/:user_id/questions' => 'passwords#questions', :as => :password_questions, via: [:get, :post]
     match '/password/:user_id/check_questions' => 'passwords#check_questions', :as => :check_password_questions, via: [:get, :post]
+    get '/email' => 'passwords#email'
```

Comment:
I don't think we should add this. If there was no route to get email before then we shouldn't add one now.

Reply:
Without that path this spec fails:

https://github.com/concord-consortium/rigse/blob/master/rails/spec/controllers/passwords_controller_spec.rb#L135

It feels like our registration system has some invalid paths through it.

Reply:
I'm testing the 'forgot password' flow on learn.staging.
A form is shown asking for an email. This is sent to:
https://learn.staging.concord.org/api/v1/passwords/reset_password
That sends me a email with a link to:
https://learn.staging.concord.org/change_password/[long hash]
With a form to fill in a new password, which posts to:
https://learn.staging.concord.org/passwords/update_users_password

So I'm pretty sure the email action on the non-api passwords control is not used anymore. You could try moving that "will send an email reset notification..." spec to the api spec (if it doesn't has one already).  And then I'd just delete the email action from the non-api passwords controller.